.github/workflows/publish.yml

```diff
@@ -1,4 +1,5 @@
-name: Publish yew & yew-macro
+name: Publish yew ONLY (WITHOUT MACRO)
```

Comment:
I guess the name should be adjusted to something like "Publish yew package(s)"

Reply:
```suggestion
name: Publish yew package(s)
```